ledger/src/bigtable_upload.rs

```diff
@@ -51,21 +51,24 @@ pub async fn upload_confirmed_blocks(
 ) -> Result<Slot, Box<dyn std::error::Error>> {
     let mut measure = Measure::start("entire upload");
 
-    info!("Loading ledger slots starting at {}...", starting_slot);
+    info!(
+        "Loading ledger slots from {} to {}",
+        starting_slot, ending_slot
+    );
```

Comment:
Since these could be getting spawned and run async, thought it would be nice to have start and end slot in here.